manim/mobject/coordinate_systems.py

```diff
@@ -119,27 +120,26 @@ def input_to_graph_point(self, x, graph):
                 return None
 
 
+@attr.s(auto_attribs=True, eq=False)
 class Axes(VGroup, CoordinateSystem):
-    CONFIG = {
-        "axis_config": {
+    axis_config: tp.Dict = {
             "color": LIGHT_GREY,
             "include_tip": True,
             "exclude_zero_from_default_numbers": True,
-        },
-        "x_axis_config": {},
-        "y_axis_config": {"label_direction": LEFT,},
-        "center_point": ORIGIN,
     }
+    x_axis_config: tp.Dict = attr.ib(default=attr.Factory(dict))
```

Comment:
Same comment about removing CONFIG later.
```suggestion
    x_axis_config: dict = attr.ib(factory=dict)
```